api/test/unit/utils/test_image_proxy.py

```diff
@@ -331,18 +331,8 @@ def test_get_successful_records_response_code(
 
 
 alert_count_params = pytest.mark.parametrize(
-    "count_start, should_alert",
-    [
-        (0, True),
-        (1, True),
-        (50, True),
-        (99, True),
-        (100, False),
-        (999, True),
-        (1000, False),
-        (1500, False),
-        (1999, True),
-    ],
+    "count_start",
+    [0, 1, 50, 99, 100, 999, 1000, 1500, 1999],
```

Comment:
I don't know if there is any reason to test so many different counts now as they all behave identically.

Reply:
Testing 0, 1, and some other higher number would be fine, I agree!

Reply:
Done in [2554558](https://github.com/WordPress/openverse/pull/3709/commits/2554558637150365326022bf2c5dce4ebc058708)!